src/events/events.jobs.controller.ts

```diff
@@ -55,4 +60,59 @@ export class EventsJobsController {
     await this.eventsService.deleteBlockMined(block);
     return { requeue: false };
   }
+
+  @MessagePattern(GraphileWorkerPattern.UPDATE_LATEST_POINTS)
+  async updateLatestPoints({
+    userId,
+    type,
+  }: {
+    userId: number;
+    type: EventType;
+  }): Promise<void> {
+    await this.prisma.$transaction(async (prisma) => {
```

Comment:
We generally don't use `prisma` in our controllers to leave DB abstractions at the service level. Could we move this aggregation to the `EventsService` and call that method here?

Reply:
Yea I can move this there